wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/ElevatorSim.java

```diff
@@ -98,66 +98,64 @@ public ElevatorSim(
    *
    * @param gearbox The type of and number of motors in the elevator gearbox.
    * @param gearing The gearing of the elevator (numbers greater than 1 represent reductions).
-   * @param carriageMassKg The mass of the elevator carriage.
-   * @param drumRadiusMeters The radius of the drum that the elevator spool is wrapped around.
-   * @param minHeightMeters The min allowable height of the elevator.
-   * @param maxHeightMeters The max allowable height of the elevator.
+   * @param carriageMass The mass of the elevator carriage in kg.
+   * @param drumRadius The radius of the drum that the elevator spool is wrapped around in meters.
+   * @param minHeight The min allowable height of the elevator.
+   * @param maxHeight The max allowable height of the elevator.
    * @param simulateGravity Whether gravity should be simulated or not.
-   * @param startingHeightMeters The starting height of the elevator.
+   * @param startingHeight The starting height of the elevator.
    * @param measurementStdDevs The standard deviations of the measurements. Can be omitted if no
    *     noise is desired. If present must have 1 element for position.
    */
   public ElevatorSim(
```

Comment:
Units are missing in the `@param` comments for `minHeight`, `maxHeight`, and `startingHeight`.